robolectric-annotations/src/main/java/org/robolectric/annotation/Config.java

```diff
@@ -102,6 +105,20 @@
   String qualifiers() default DEFAULT_QUALIFIERS;
 
   /**
+   * Android build type to use for resource resolution, such as "debug".
+   *
+   * @return Build type used for resource resolution.
+   */
+  String buildType() default DEFAULT_BUILD_TYPE;
+
+  /**
+   * Android product flavor to use for the resource resolution, such as "demo".
+   *
+   * @return Flavor used for resource resolution.
+   */
+  String flavor() default DEFAULT_FLAVOR;
```

Comment:
Hey Jake!

Huh, interesting, didn't know about that. What would you think of using hyphen-separated `String flavors` (like `String qualifiers`), so we could reserve `String[] flavors(es?)` for multi test run configs, as sdk is used now?